tests/block_tools.py

```diff
@@ -266,6 +275,7 @@ async def new_plot(
         args.nobitfield = False
         args.exclude_final_dir = False
         args.list_duplicates = False
+        args.exclude_final_dir = exclude_final_dir
```

Comment:
I don't see the `args.exclude_final_dir` field used anywhere

Reply:
`args` is passed here https://github.com/Chia-Network/chia-blockchain/blob/5dc87472983816b054f1bfa7970b9301814c3432/tests/block_tools.py#L290-L296

and `exclude_final_dir` is used here 

https://github.com/Chia-Network/chia-blockchain/blob/5dc87472983816b054f1bfa7970b9301814c3432/chia/plotting/create_plots.py#L236-L244